spring-kafka/src/main/java/org/springframework/kafka/support/AbstractKafkaHeaderMapper.java

```diff
@@ -191,6 +193,16 @@ public void addRawMappedHeader(String name, boolean toString) {
 		this.rawMappedHeaders.put(name, toString);
 	}
 
+	/**
+	 * Add patterns for matching multi-value headers under the same key.
+	 * @param patterns the patterns for header.
+	 */
+	public void addHeaderPatternsForMultiValue(String ... patterns) {
```

Comment:
This is not correct name from Java Beans specification perspective.
Really has to be `setMultiValueHeaderPatterns`.

Not sure why you use `for` over there.